File: examples/python/search_1.py
def linear_search(arr, target):
    for i, x in enumerate(arr):
        if x == target:
            return i
    return -1


if __name__ == "__main__":
    data = [3, 7, 2, 9, 1]
    print(linear_search(data, 7))
    print(linear_search(data, 5))

File: examples/javascript/search_1.js
function linearSearch(arr, target) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === target) return i;
  }
  return -1;
}

const arr = [3, 7, 2, 9, 1, 5];
console.log(linearSearch(arr, 9));
console.log(linearSearch(arr, 4));
